easybuild/easyconfigs/l/LAMMPS/LAMMPS-20180316-goolf-1.7.20.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "MakeCp"
+
+name = "LAMMPS"
+version = "20180316"
+
+homepage = 'http://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code,
+and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/lammps/lammps/archive/']
+sources = ['stable_16Mar2018.tar.gz']
+
+builddependencies = [
+    ('Python', '2.7.11'),
+    ('libpng', '1.6.21'),
+]
+
+dependencies = [
+    ('Python', '2.7.11'),
```

Comment:
@VilleS1 You have `Python` listed both a build dependency and (runtime) dependency, that's why the tests are failing.